test/run_cli.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const ChildProcess = require('child_process');
+const Path = require('path');
+
+// Declare internals
+
+const internals = {
+    labPath: Path.join(__dirname, '..', 'bin', '_lab')
+};
+
+module.exports = (args, callback) => {
+
+    const cli = ChildProcess.spawn('node', [].concat(internals.labPath, args));
+    let output = '';
+    let errorOutput = '';
+    let combinedOutput = '';
+
+    cli.stdout.on('data', (data) => {
+
+        output += data;
+        combinedOutput += data;
+    });
+
+    cli.stderr.on('data', (data) => {
+
+        errorOutput += data;
+        combinedOutput += data;
+    });
+
+    cli.once('close', (code, signal) => {
+
+    	if (signal) {
+    		callback(new Error('Unexpected signal: ' + signal));
```

Comment:
Its worse than you thought: tabs!

<img src="http://knitschtick.com/wp-content/uploads/2014/03/heyo.jpeg" />